### PR TITLE
Add sha256 checksum of configmaps to each deployments / Add values-xxx.yaml

### PR DESCRIPTION
**Pull request by rinx**
### Description:

This PR includes the followings.
- Add annotations of configmaps' sha256 checksum
- Add values-dev.yaml, values-cassandra.yaml, values-scylla.yaml

### Related Issue:

Nothing.

### How Has This Been Tested?:

Nothing.

### Environment:

- Golang Version: 1.13.8
- Docker Version: 19.03.5
- Kubernetes Version: 1.16.3
- NGT Version: 1.8.4

### Types of changes:

- [ ] Bug fix [type/bug]
- [X] New feature [type/feature]
- [ ] Add tests [type/test]
- [ ] Security related changes [type/security]
- [ ] Add documents [type/documentation]
- [ ] Refactoring [type/refactoring]
- [ ] Update dependencies [type/dependency]
- [ ] Update benchmarks and performances [type/bench]
- [ ] Update CI [type/ci]

### Changes to Core Features:

- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?

### Checklist:

- [X] I have read the [CONTRIBUTING](https://github.com/vdaas/vald/blob/master/CONTRIBUTING.md) document.
- [X] I have checked open [Pull Requests](https://github.com/vdaas/vald/pulls) for the similar feature or fixes?
- [ ] I have added tests and benchmarks to cover my changes.
- [ ] I have ensure all new and existing tests passed.
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: vald/templates/agent/daemonset.yaml [hunk 1]
```diff
@@ -20,6 +20,8 @@ metadata:
   name: {{ .Values.agent.name }}
   labels:
     app: {{ .Values.agent.name }}
+  annotations:
+    checksum/configmap: {{ include (print $.Template.BasePath "/agent/configmap.yaml") . | sha256sum }}
```
Comment (rinx) [inline vald/templates/agent/daemonset.yaml]: badge
it is required to discuss about the agents should have sha256 checksum or not.

Comment (kpango) [reply vald/templates/agent/daemonset.yaml]: for now agent does not have their own scheduler, if we change configmap agent pod will be restart it means agent index will be flush.
we should disable this annotation until we impl agent scheduler

Comment (rinx) [reply vald/templates/agent/daemonset.yaml]: okay. i've just commented out these annotations.

**Pull request merged**